install.sh

```
@@ -33,6 +22,761 @@ print_message "This script will install BirdNET-Go and its dependencies." "$YELL
+        return 1
+    fi
+    
+    log_message "INFO" "=== Capturing Current Image State ($context) ==="
+    
+    # Try to get running container image first
+    local current_image=""
+    local image_hash=""
+    local image_tag=""
+    
+    # Check for running BirdNET-Go container
+    local running_container=$(safe_docker ps --filter "name=birdnet-go" --format "{{.Image}}" 2>/dev/null | head -1)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)